assets/js/components/wp-dashboard/wp-dashboard-clicks.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * WPDashboardClicks component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../modules/search-console/datastore/constants';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { extractSearchConsoleDashboardData, isZeroReport } from '../../modules/search-console/util';
+import { trackEvent } from '../../util/tracking';
+import DataBlock from '../data-block';
+import PreviewBlock from '../PreviewBlock';
+import ReportError from '../ReportError';
+import ReportZero from '../ReportZero';
+const { useSelect } = Data;
+
+const WPDashboardClicks = () => {
+	const { data, error, loading } = useSelect( ( select ) => {
+		const store = select( STORE_NAME );
+
+		const commonArgs = {
+			dateRange: select( CORE_USER ).getDateRange(),
+		};
+
+		const args = {
+			dimensions: 'date',
+			compareDateRanges: true,
+			...commonArgs,
+		};
```

Comment:
Huh, I just noticed this one too. Why assign the `commArgs` variable here? It's not used anywhere except to be placed immediately into the dictionary below. Let's add it to the regular `args` variable instead:
```suggestion
		const args = {
			compareDateRanges: true,
			dateRange: select( CORE_USER ).getDateRange(),
			dimensions: 'date',
		};
```